python/dgl/distributed/client.py

```diff
@@ -2,33 +2,155 @@
 
 """Functions used by client."""
 
-def connect_to_server(server_namebook):
+from . import rpc
+
+import os
+import time
+import socket
+
+if os.name != 'nt':
+    import fcntl
+    import struct
+
+def local_ip4_addr_list():
+    """Return a set of IPv4 address
+    """
```

Comment:
The function is not supported in WIN since `struct` is not imported. Please raise more informative errors.